tests/wpt/tests/tools/wptrunner/wptrunner/executors/executorservodriver.py

```diff
@@ -31,6 +31,11 @@ def set_prefs(self, prefs):
         def reset_prefs(self, *prefs):
             body = {"prefs": list(prefs)}
             return self.session.send_session_command("POST", "servo/prefs/reset", body)
+        
+        def shutdown(self):
+            body = {}
+            return self.session.send_session_command("DELETE", "servo/shutdown", body)
+        
```

Comment:
```
./wptrunner/wptrunner/executors/executorservodriver.py:34:1: W293 blank line contains whitespace
./wptrunner/wptrunner/executors/executorservodriver.py:38:1: W293 blank line contains whitespace
```
Per the CI failures in https://community-tc.services.mozilla.com/tasks/FisJG5vlTpqDDj8LB1U-Iw/runs/0/logs/live/public/logs/live.log